packages/components/theme/src/components/alert.ts

```diff
@@ -42,31 +42,41 @@ const baseStyle = definePartsStyle({
   },
 })
 
-function getBg(props: StyleFunctionProps): string {
+function getBg(props: StyleFunctionProps) {
   const { theme, colorScheme: c } = props
-  const lightBg = getColor(theme, `${c}.100`, c)
   const darkBg = transparentize(`${c}.200`, 0.16)(theme)
-  return mode(lightBg, darkBg)(props)
+  return {
+    light: { [$bg.variable]: `colors.${c}.100` },
+    dark: { [$bg.variable]: darkBg },
+  }
 }
```

Comment:
Can we simplify this to reduce the spreads in the code below

```suggestion
  return {
    light:  `colors.${c}.100`,
    dark: darkBg,
  }
}
```